ST_mortality.c

```diff
@@ -1064,3 +1195,98 @@ void _kill_maxage(void) {
         }
     }
 }
+
+/**
+ * \brief Converts the biomass of cheatgrass to the % cover of cheatgrass.
+ * 
+ * This relationship between biomass and percent cover was derived by Maggie 
+ * England.
+ * 
+ * \param biomass is the biomass of cheatgrass.
+ * 
+ * \return A double between 0 and 1 representing the percent of the plot 
+ *         covered by cheatgrass.
```

Comment:
@chaukap this is confusing to me. Percent cover ranges from 0 to 100, not 0 and 1 - see the spreadsheet in https://github.com/DrylandEcology/STEPWAT2/issues/324. Can you clarify here why this is done?

Reply:
@kpalmqui This was my mistake, commit 217f2c3 fixes it.